pkg/controllers/vdb/createdb_reconciler.go

```diff
@@ -115,7 +115,8 @@ func (c *CreateDBReconciler) execCmd(ctx context.Context, initiatorPod types.Nam
 
 // preCmdSetup will generate the file we include with the create_db.
 // This file runs any custom SQL for the create_db.
-func (c *CreateDBReconciler) preCmdSetup(ctx context.Context, initiatorPod types.NamespacedName, podList []*PodFact) (ctrl.Result, error) {
+func (c *CreateDBReconciler) preCmdSetup(ctx context.Context, initiatorPod types.NamespacedName,
+	initiatorIP string, podList []*PodFact) (ctrl.Result, error) {
```

Comment:
Where is `initiatorIP` used in `preCmdSetup`?

Reply:
`preCmdSetup` is part of an interface, so we need the parm. If we upgrade golangci-lint, we can enabled a linter that will pick this thing up. They recommend using `_` as the variable name. I can rename the parameter to that and upgrade the linter as a follow-on task.